### DISCUSSION TITLE
What is  inputSequenceMetafiles

### DISCUSSION THREAD

**Discussion by 1939938853** [PlusToolkit/PlusLib#946] [Q&A]
Hello,

On the PlusLib `vtkSpacingCalibAlgoTest.cxx ` file, there it requires an input file defined as `std::vector<std::string> inputSequenceMetafiles`. So what is  inputSequenceMetafiles? Any example on it? Or the Sequence Meta files are just a sequence of files like "fCal_Test_Calibration_3NWires.igs.mha"?

Thank you!

**Comment by adamrankin** [comment]
The file you've described is a sequence metafile. Files ending in .igs.mha, .igs.mhd, .igs.nrrd are sequence metafiles.

**Reply by 1939938853** [reply]
Thanks
